pod-operation/src/components/lim_temperature.rs

```diff
@@ -6,6 +6,19 @@ use ads1x1x::{Ads1x1x, DynamicOneShot, SlaveAddr};
 use nb::block;
 use rppal::i2c::I2c;
 
+const DIVIDER_RESISTANCE: f32 = 22000.0;
+const VCC: f32 = 5.0;
+const BETA: f32 = 3950.0;
+const R_0: f32 = 10000.0;
+const ROOM_TEMP: f32 = 298.15;
+
+fn voltage_to_temp(voltage: i16) -> f32 {
+	let voltage = f32::from(voltage) / 1000.0;
+	let thermistor_resistance: f32 = (voltage * DIVIDER_RESISTANCE) / (VCC - voltage);
+	let r_inf = R_0 * std::f32::consts::E.powf(-BETA / ROOM_TEMP);
```

Comment:
Thought: `r_inf` could be a constant value, but `f32::powf` isn't a `const fn`, so we won't be able to make this `const`. However, I think the compiler is smart enough to recognize this optimization.